shepherd_bms/src/main.cpp

```diff
@@ -280,7 +290,35 @@ void shepherdMain()
 		digitalWrite(CHARGE_SAFETY_RELAY, LOW);
 	}
 
-	compute.sendMCMsg(accData->chargeLimit, accData->dischargeLimit);
+	//Transitioning out of boost
```

Comment:
not important to code but noting here anyway that we should put boosting state diagram --> confluence